packages/client/src/clients/guide/client.ts

```diff
@@ -470,6 +518,42 @@ export class KnockGuideClient {
     });
   }
 
+  exitDebugMode() {
```

Comment:
Adding this method here so that the GuideToolbar doesn't have to be concerned with clearing the url/local storage/etc.